index.bs

```diff
@@ -3189,6 +3189,10 @@ the value of a <dfn id="k-rate">k-rate</dfn> parameter MUST be sampled at the ti
 very first sample-frame, and that value MUST be used for the entire
 block. <dfn id="a-rate">a-rate</dfn> parameters MUST be sampled for
 each sample-frame of the block.
+For many nodes, the rate of the {{AudioParam}} can be selected by
```

Comment:
many? some? Not sure.

Reply:
Don't know how best to phrase this.

Reply:
Something like:

> Depending on the `AudioParam` its rate can be controlled via setting the attribute, or it can be fixed, see each `AudioParam`s for details.